breeze-complete

```diff
@@ -44,49 +44,7 @@ EOF
 _BREEZE_ALLOWED_STATIC_CHECKS=$(cat <<-EOF
 all
 all-but-pylint
-airflow-config-yaml
-base-operator
-bat-tests
-build
-build-providers-dependencies
-check-apache-license
-check-executables-have-shebangs
-check-hooks-apply
-check-integrations
-check-merge-conflict
-check-xml
-consistent-pylint
-daysago-import-check
-debug-statements
-detect-private-key
-doctoc
-end-of-file-fixer
-fix-encoding-pragma
-flake8
-forbid-tabs
-incorrect-use-of-LoggingMixin
-insert-license
-isort
-language-matters
-lint-dockerfile
-mixed-line-ending
-mypy
-provide-create-sessions
-pydevd
-pydocstyle
-pylint
-pylint-tests
-python-no-log-warn
-rst-backticks
-setup-order
-shellcheck
-stylelint
-trailing-whitespace
-update-breeze-file
-update-extras
-update-local-yml-file
-update-setup-cfg-file
-yamllint
+$(grep -E "id *:" .pre-commit-config.yaml | cut -d ":" -f 2 | sed "s/^ //" | sort | uniq)
```

Comment:
You never know which directory you are in when you source .breeze-complete so this one will not always work. ./breeze-complete should be fully standalone and should not include nor parse any other data.

I think this should be a pre-commit to update those values rather than grep run every time you source it.